simulation.py

```diff
@@ -277,23 +277,29 @@ def epsilon_to_png(self):
                     h5_file=self.eps_file + ':data', 
                     output_file=defaults.temporary_epsh5)
         log.info("calling: {0}".format(callstr))
-        if not sp.call(callstr.split(), cwd=self.workingdir):
-            # no error, continue:
-            dct = dict(self.__dict__, h5_file=defaults.temporary_epsh5)
-            # save dielectric to png:
-            if self.geometry.is3D:
-                callstr = (defaults.epsh5topng_call_3D % dct,
-                           defaults.epsh5topng_call_3D_cross_sect % dct)
-            else:
-                callstr = (defaults.epsh5topng_call_2D % dct,)
-            retcode = 0
-            for s in callstr:
-                if not retcode:
-                    log.info("calling: {0}".format(s))
-                    retcode = retcode or sp.call(s.split(), 
-                                                 cwd=self.workingdir)
-        else:
+        
+        try:
+            sp.call(callstr.split(), cwd=self.workingdir)
+        except OSError as err:
+            log.warning('The following OSError occurred while calling mpb:' +
+                        '\n\t{}\n'.format(err) + 
+                        '\tMaybe mpb-data is not properly installed.')
```

Comment:
Sorry, I added to my comment while you were already submitting again. Here it is again:

OK, I just read the code above... ;) Just before this statement, the logs contains the line "calling ..." and the actually failed command. So no need to be so elaborate here, just put something like: "Command could not be executed. Will continue without converting epsilon.h5 to png."

By the way, it is intended that some user can put any command in defaults.py, so could even change to something different than mpb-data.